### DISCUSSION TITLE
How does Quarto install Pandoc?

### DISCUSSION THREAD

**Discussion by Adam-Antios** [quarto-dev/quarto-cli#9562] [Q&A]
### Description

I have installed Quarto, but I try and invoke "pandoc file.md -o out.html" in the terminal, pandoc is not available. How does Quarto install Pandoc? Isn't it available system-wide?

**Comment by cscheid** [comment]
Quarto bundles its own version internally. You can access it via `quarto pandoc file.md -o out.html`
